protopipe/pipeline/utils.py

```diff
@@ -3,15 +3,15 @@
 import argparse
 
 import matplotlib.pyplot as plt
+import os.path as path
 
-
-def save_fig(path, outdir, name, fig = None):
+def save_fig(outdir, name, fig = None):
     """Save a figure in multiple formats."""
     for ext in ["pdf", "png"]:
```

Comment:
not the purpose of this PR I know, but why `pdf` and `png`?

Reply:
PDF is "legacy", "png" was better for wikipages like Redmine.
Of course, this can change in the future if someone brings in particular needs.

Reply:
Thanks for the info.